### PR TITLE
feat: add connection details in stats for nerds

### PR DESCRIPTION
**Pull request by hdz-666**
### Details(context, link the issue, how was the bug fixed, what does the new feature do)
image

- Updated sdk and hmsSdkactions also updated interface to have call details 
- Rendered new call details accordion on stats for nerds modal
- added a height for stats for nerds to handle overflow

### CODE REVIEW

**Comment by KaustubhKumar05** [comment]
@hdz-666  can you cherry-pick and create a new branch from dev?

**Review by KaustubhKumar05 — DISMISSED** [review]

File: packages/roomkit-react/src/Prebuilt/components/StatsForNerds.jsx [hunk 1]
```diff
@@ -46,7 +50,7 @@ export const StatsForNerds = ({ onOpenChange }) => {
         <Dialog.Content
           css={{
             width: 'min(500px, 95%)',
-            maxHeight: '100%',
+            height: 'min(656px, 90%)',
```
Comment (hdz-666) [inline packages/roomkit-react/src/Prebuilt/components/StatsForNerds.jsx:53]: copied from settings Desktop modal might need to reconsider this for mobile

File: packages/hms-video-store/src/schema/settings.ts [hunk 2]
```diff
@@ -6,3 +6,9 @@ export interface HMSMediaSettings {
   audioOutputDeviceId?: string;
   audioMode?: HMSAudioMode;
 }
+
+export interface CallDetails {
```
Comment (raviteja83) [inline packages/hms-video-store/src/schema/settings.ts:10]: DebugInfo

File: packages/roomkit-react/src/Prebuilt/components/StatsForNerds.jsx [hunk 3]
```diff
@@ -229,8 +236,29 @@ const TrackStats = ({ trackID, layer, local }) => {
...
 };
 
-const StatsRow = React.memo(({ label, value }) => (
-  <Box css={{ bg: '$surface_bright', w: 'calc(50% - $6)', p: '$8', r: '$3' }}>
+const CallDetails = ({ details }) => {
+  return (
+    <Accordion.Root type="single" collapsible css={{ w: '100%' }}>
+      <Accordion.Item value="Call Details">
+        <Accordion.Header>
+          <Label variant="body2" css={{ c: '$on_surface_high' }}>
```
Comment (raviteja83) [inline packages/roomkit-react/src/Prebuilt/components/StatsForNerds.jsx:244]: this can be at the bottom and default can be closed accordion

Comment (hdz-666) [reply packages/roomkit-react/src/Prebuilt/components/StatsForNerds.jsx:244]: default is closed only

Comment (hdz-666) [reply packages/roomkit-react/src/Prebuilt/components/StatsForNerds.jsx:244]: moving

File: packages/hms-video-store/src/sdk/index.ts [hunk 4]
```diff
@@ -279,6 +280,17 @@ export class HMSSdk implements HMSInterface {
     return this.transport?.getWebrtcInternals();
   }
 
+  getDebugInfo(): DebugInfo {
+    const websocketUrl = this.transport.getWebsocketEndpoint();
+    const enabledFlags = Object.values(InitFlags).filter(flag => this.transport.isFlagEnabled(flag));
+    const initEndpoint = this.store.getConfig()?.initEndpoint;
+    return {
+      websocketURL: websocketUrl,
```
Comment (KaustubhKumar05) [inline packages/hms-video-store/src/sdk/index.ts:288]: Nit, we can keep this consistent too

**Review by KaustubhKumar05 — APPROVED** [review]

**Review by KaustubhKumar05 — DISMISSED** [review]

File: packages/roomkit-react/src/Prebuilt/components/StatsForNerds.jsx [hunk 5]
```diff
@@ -229,8 +237,29 @@ const TrackStats = ({ trackID, layer, local }) => {
...
+    <Accordion.Root type="single" collapsible css={{ w: '100%' }}>
+      <Accordion.Item value="Debug Info">
+        <Accordion.Header>
+          <Label variant="body2" css={{ c: '$on_surface_high' }}>
+            Debug Info
+          </Label>
+        </Accordion.Header>
+        <Accordion.Content>
+          <Flex css={{ flexWrap: 'wrap', mt: '$10', gap: '$10' }}>
+            <StatsRow css={{ w: 'calc(100%)' }} label="Websocket URL" value={details?.websocketURL} />
```
Comment (raviteja83) [inline packages/roomkit-react/src/Prebuilt/components/StatsForNerds.jsx:251]: calc is not needed i guess

**Review by raviteja83 — APPROVED** [review]

**Review by ygit — APPROVED** [review]
LGTM

**Pull request merged**